tests/google_sheets/test_google_sheets_pipeline.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from pipelines.google_sheets import google_sheets
+from tests.utils import ALL_DESTINATIONS, assert_load_info
+
+
+TEST_SPREADSHEETS = []
+
+
+@pytest
+def test_authentication():
+    """"
+    Just tests the authentication to the Google Sheets API
+    """
+    assert 1 == 1
+
+
+@pytest.mark.parametrize('spreadsheet_id', TEST_SPREADSHEETS)
+def test_spreadsheet_access():
+    """"
+    Just tests if access to the spreadsheets is ok
+    """
+    assert 1 == 1
+
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_appending():
+    """"
+    Test that adding new data in the sheets will add new data to the destinations
+    """
+    assert 1 == 1
+
+
+@pytest.mark.parametrize('spreadsheet_id', TEST_SPREADSHEETS)
+def test_all_ranges_loaded():
+    """"
+    Test that all ranges are loaded into a database
+    """
+    assert 1 == 1
+
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_full_load():
+    """"
+    Full run of a simple pipeline with all destinations
+    """
+    assert 1 == 1
+
```

Comment:
we should test
1. a range that contains all possible sheet data types
2. a range that has only header
3. a range that has only one row that is not header (just single line of data)
4. a range that has some empty cells - for various data types
5. a range that has inconsistent data types ie. a date in first row and then something else (number, string etc.)
6. a range that has whole empty row
7. a range that has many empty rows
8. a range that has two "tables" with data like here https://docs.google.com/spreadsheets/d/11G95oVZjieRhyGqtQMQqlqpxyvWkRXowKE8CtdLtFaU/edit#gid=55249181